package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@xenit/redux-utils",
+  "name": "@vierbergenlars/redux-utils",
```

Comment:
Should this not just stay the same as it was?

Reply:
Eh, yes.
Good catch.
It's a leftover from when I was testing in a private repository :sweat: